### PR TITLE
change quotes to allow literal ds-token types

### PR DESCRIPTION
**Pull request by andnorda**
Ref. https://nav-it.slack.com/archives/C7NE7A8UF/p1696241813007549

### CODE REVIEW

**Review by karl-run — DISMISSED** [review]
wow very nice

File: @navikt/core/tokens/config/config.js [hunk 1]
```diff
@@ -36,6 +36,9 @@ module.exports = {
       ],
     },
     ts: {
+      options: {
+        outputStringLiterals: true,
+      },
```
Comment (karl-run) [inline @navikt/core/tokens/config/config.js]: den skal forresten under den `format: "typescript/es6-declarations",` arrayet under

Comment (andnorda) [reply @navikt/core/tokens/config/config.js]: Aha! Leste docen litt raskt, og dette funka.

Flytta den nå.

**Review by KenAJoh — APPROVED** [review]
💯

**Pull request merged**